scene/resources/dynamic_font.h

```diff
@@ -61,7 +61,7 @@ class DynamicFontData : public Resource {
 		};
 		bool operator<(CacheID right) const;
 		CacheID() {
-			key = 0;
+			memset(this, 0, sizeof(CacheID));
```

Comment:
It would be cleaner to initialize the members explicitly instead of using `memset`. This would break if we add a member that needs a non-0 initialization. CC @hpvb

Reply:
Thanks for the review. I've made the proper fix suggested by @reduz and it works perfectly well on my end.